with-better-auth/app/(auth)/sign-in.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+import { TextInput, Button, ScrollView, Text, StyleSheet } from "react-native";
+import { authClient } from "@/lib/auth-client";
+import { Link } from "expo-router";
+
+export default function SignIn() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleLogin = async () => {
+    await authClient.signIn.email({
+      email,
+      password,
+    });
```

Comment:
Would it make sense to handle errors in this example?

```suggestion
const signInResponse = await authClient.signIn.email({
      email,
      password,
    });

    if (signInResponse.error) {
      Alert.alert("Error", signInResponse.error.message);
      return;
    }
```